Cython/Utility/CythonFunction.c

```diff
@@ -627,6 +660,16 @@ static CYTHON_INLINE PyObject *__Pyx_CyFunction_Call(PyObject *func, PyObject *a
 static PyObject *__Pyx_CyFunction_CallAsMethod(PyObject *func, PyObject *args, PyObject *kw) {
     PyObject *result;
     __pyx_CyFunctionObject *cyfunc = (__pyx_CyFunctionObject *) func;
+
+#if CYTHON_METH_FASTCALL
+    /* Prefer vectorcall if available. This is not the typical case, as
+     * CPython would normally use vectorcall directly instead of tp_call. */
```

Comment:
Seems worth discarding this comment from the generated code files by prefixing it with `//`.

Reply:
I always forget this...